### ISSUE TITLE
Conflict between colored output by a target command and viddy's colored diff

### ISSUE THREAD

**Issue by nobuto-m** [sachaos/viddy#43]
Here is an example output. When a target command has colored output (e.g. `grep --color=always`) and diff highlighting with viddy are enabled at the same time, a color code (`[[0m[01;38;5;202m[0m[K`) is visible in the terminal and it can break indents.

```
Every───┐┌Command───────────────────────────────────────────────────────────────────────────────────────────┐┌Time─────────[0/988]
│2s      ││ps aux | sort -R | grep --color=always [k]worker                                                  ││2022-06-06 07:16:06│
└────────┘└──────────────────────────────────────────────────────────────────────────────────────────────────┘└───────────────────┘
root         708  0.0  0.0      0     0 ?        I<   Jun05   0:02 [kworker/0:1H-kblockd[]                                         
root     1715525  0.0  0.0      0     0 ?        I    07:09   0:00 [[0m[01;38;5;202m[0m[Kkworker[0m[m[0m[K/6:10-events[]           
root     1759911  0.0  0.0      0     0 ?        I    07:14   0:00 [kworker/10:1-events[]                                          
root     1599407  0.0  0.0      0     0 ?        I    07:06   0:00 [kworker/15:3-cgroup_destroy[]                                  
root     1619560  0.1  0.0      0     0 ?        I    07:07   0:00 [kworker/0:11-events[]
```

**Comment by sachaos** [comment]
I think this bug fixed in viddy v1.0.0-rc.1 .

https://github.com/sachaos/viddy/issues/117#issuecomment-2291069041

**Comment by sachaos** [comment]
Viddy v1.0.0 is available now.
